src/main/java/appeng/core/AppEng.java

```diff
@@ -141,7 +141,7 @@ private void preInit( final FMLPreInitializationEvent event )
 		final File recipeFile = new File( this.configDirectory, "CustomRecipes.cfg" );
 		final Configuration recipeConfiguration = new Configuration( recipeFile );
 
-		AEConfig.instance = new AEConfig( configFile );
+		AEConfig.init( configFile );
 		FacadeConfig.instance = new FacadeConfig( facadeFile );
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make the enclosing method "static" or remove this set. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2696)